### PR TITLE
Rewrite the indexing API to not use GraphQL queries internally

### PR DESCRIPTION
**Pull request by lutter**
This also adds `entityCount` as a value that is accessible through the API

### CODE REVIEW

File: graph/src/data/subgraph/status.rs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        self.0
+    }
+}
+
+impl From<EthereumBlock> for q::Value {
+    fn from(block: EthereumBlock) -> Self {
+        object! {
+            __typename: "EthereumBlock",
+            hash: block.0.hash_hex(),
+            number: format!("{}", block.0.number),
```
[the remark this answers is not in the document]
Comment (lutter) [reply graph/src/data/subgraph/status.rs:33]: Out of curiosity, what's the advantage? I thought `format!` calls `to_string` implicitly.

[the remark this answers is not in the document]
Comment (leoyvens) [reply graph/src/data/subgraph/status.rs:33]: I don't think there is any performance advantage, just a code style choice, up to you.

[the remark this answers is not in the document]
Comment (lutter) [reply graph/src/data/subgraph/status.rs]: I'd rather stick with the implicit call of `to_string` then.

File: graph/src/data/subgraph/status.rs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    }
+}
+
+impl From<Vec<Info>> for Infos {
+    fn from(infos: Vec<Info>) -> Self {
+        Infos(infos)
+    }
+}
+
+impl From<Infos> for q::Value {
```
[the remark this answers is not in the document]
Comment (lutter) [reply graph/src/data/subgraph/status.rs]: Good point. I just added a commit that cleans the serialization up by using `IntoValue` for everything in that module

File: store/postgres/src/detail.rs [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+        ethereum_head_block_number -> Nullable<Numeric>,
+        network -> Text,
+        node_id -> Nullable<Text>,
+        // We don't map block_range
+        // block_range -> Range<Integer>,
+    }
+}
+
+table! {
+    subgraphs.subgraph_error (vid) {
```
[the remark this answers is not in the document]
Comment (lutter) [reply store/postgres/src/detail.rs:49]: Yep; I just hope it doesn't lead to huge merge conflicts with the sharded store implementation.

**Comment by lutter** [comment]
Addressed all review comments

**Pull request merged**